src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -182,6 +197,20 @@ private static TargetConfiguration getTargetConfiguration(Triplet targetTriplet)
         }
     }
 
+    /*
+     * check if this host can be used to provide binaries for this target.
+     * host and target should not be null.
+     */
+    private static boolean isMatchingConfiguration (Triplet host, Triplet target) {
```

Comment:
The name of method is not matching to what it a actually does. Too generic i think. What about something like canProvideBinaries?

Reply:
I'll change it in "canCompileTo" as that is indeed what it does. We want to check if the host can compile to target. (provideBinaries is more generic/broad)